applications/GeoMechanicsApplication/custom_workflows/dgeosettlement.cpp

```diff
@@ -379,7 +379,7 @@ void KratosGeoSettlement::PrepareModelPart(const Parameters& rSolverSettings)
     }
 
     // Note that the computing part and the main model part _share_ their process info and properties
-    GetComputationalModelPart().SetProcessInfo(main_model_part.GetProcessInfo());
```

Comment:
please note that this is a behaviour change. Before it was a copy, now it is a pointer assignement... @KratosMultiphysics/geomechanics do you agree with this?

Reply:
hmmm.. I'm not sure what you meant by you were wrong and which previous comment you're referring to. The one above or another recent comment.

Because it was indeed a copy before and not a pointer assignment. However, the comment on line 382:
> Note that the computing part and the main model part _share_ their process info and properties
does make it seem like it does indeed intended to be a pointer assignment, and is in line with the change that I made. 

Can someone verify? @KratosMultiphysics/geomechanics